donkeycar/parts/actuator.py

```diff
@@ -915,19 +887,18 @@ def shutdown(self):
 
     
 #
-# This is being replaced by pins.py and PulseController.  
-# GPIO pins can be configured using RPi.GPIO or PIGPIO, 
+# This is being replaced by pins.py and PulseController.
+# GPIO pins can be configured using RPi.GPIO or PIGPIO,
 # so this is redundant
 #
 @deprecated("This will be removed in a future release in favor of PulseController")
 class RPi_GPIO_Servo(object):
     '''
     Servo controlled from the gpio pins on Rpi
     '''
-    def __init__(self, pin, freq = 50, min=5.0, max=7.8):
-        import RPi.GPIO as GPIO
+    def __init__(self, pin, pin_scheme=GPIO.BCM, freq = 50, min=5.0, max=7.8):
```

Comment:
Why do you use frequency 50 here, whereas we use 60 everywhere else for servos, ESCs?

Reply:
I don't use 50 there.  That is legacy code that will eventually be removed.

Reply:
Ok, thanks you're right.